code/datums/components/extradimensional_storage.dm

```diff
@@ -1,6 +1,7 @@
 /datum/component/extradimensional_storage
```

Comment:
Should probably be marked as abstract, right? Or am I missing something? At a glance it feels like no one should be instantiating this type.